### PR TITLE
Updated setup instructions

### PR DESCRIPTION
**Pull request by jedireza**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,22 @@
...
-## Usage
 
-Run `npm install` from the root, then run:
+## Setup
+
+```bash
+$ mv secrets.example.js secrets.js
+```
+
+Now you can customize `/secrets.js` or use environment variables for config.
```
Comment (redonkulus) [inline README.md]: We should be clearer about what this is used for. Basically mention that this is used for Github API access and that the user would need to create an application on Github. Provide a link to Githubs' docs on this process. Then mention that you need to fill in the client_id and client_secret in this file.

**Comment by redonkulus** [comment]
:shipit:

**Comment by jedireza** [comment]
FYI: removed the old github api notes

**Comment by redonkulus** [comment]
:shipit:

**Pull request merged**